### PR TITLE
No more Guava in Kotlin code

### PR DESCRIPTION
**Pull request by jrodbx**
Next up: Konversion

### CODE REVIEW

**Comment by swankjesse** [comment]
I’d like to consider keeping Guava just for Multimap. You can kind of fake it with `getAndPut()` but it’s clumsy, and especially clumsy to do things like deep immutability on ’em.

**Comment by jrodbx** [comment]
I see.  I thought the goal was to rid the dependency.  Should we just copy that code into this project?  On the other hand, if MultiMap usage is limited to what's changed here, maybe it's OK to migrate to getAndPut()?

**Comment by swankjesse** [comment]
I don’t feel too strongly. Lets proceed with what you have here.

But if later we need a multimap, I think it’s probably worthwhile to bring it back! The dependency doesn’t hurt.

File: wire-schema/src/main/java/com/squareup/wire/schema/Options.kt [hunk 1]
```diff
@@ -171,37 +160,40 @@ class Options(
...
+      is List<*> -> a + b as List<*>
 
       is Map<*, *> -> {
         @Suppress("UNCHECKED_CAST") // All maps have this type.
         union(linker, a as Map<ProtoMember, Any?>, b as Map<ProtoMember, Any>)
       }
 
-      else -> {
-        linker.addError("conflicting options: %s, %s", a, b)
-        a // Just return any placeholder.
```
Comment (oldergod) [inline wire-schema/src/main/java/com/squareup/wire/schema/Options.kt]: Not needed?

**Pull request merged**